extensions/cpsection/language/view.py

```diff
@@ -132,16 +134,20 @@ def _add_row(self, locale_code=None):
         language_palette = []
         key_list = self._language_dict.keys()
         for language_key in sorted(key_list):
+            locale = Locale(self._country_dict[language_key][0][0][:2])
```

Comment:
It was a workaround to obtain the language code from the language name (i.e. 'es' from 'spanish').
Now I created a dictionary with all the codes (Just for clarity).